app/workers/github_repos/clear_http_cache_worker.rb

```diff
@@ -0,0 +1,24 @@
+module GithubRepos
+  class ClearHttpCacheWorker
+    include Sidekiq::Worker
+    GITHUB_REPOS_URL = "https://api.github.com/user/repos?per_page=100".freeze
```

Comment:
I tried to figure out a way I could get the URL for the repos endpoint through the Oktokit gem but it is created in the request so I hardcoded it here. I know its not ideal but its the best way I could think to handle this.